Source/WebCore/css/StyleProperties.cpp

```diff
@@ -349,13 +351,13 @@ String StyleProperties::getPropertyValue(CSSPropertyID propertyID) const
     case CSSPropertyPageBreakAfter:
     case CSSPropertyPageBreakBefore:
         return pageBreakValue(shorthand);
+    case CSSPropertyPerspectiveOrigin:
+        return makeString(getPropertyValue(CSSPropertyPerspectiveOriginX), ' ', getPropertyValue(CSSPropertyPerspectiveOriginY));
+    case CSSPropertyTransformOrigin:
+        return makeString(getPropertyValue(CSSPropertyTransformOriginX), ' ', getPropertyValue(CSSPropertyTransformOriginY));
```

Comment:
What about perspective-origin-z ? It should be omitted if equal to 0, but specified otherwise

Reply:
Yes, fixing this right now.